src/rulesets/oas3/schemas/main.json

```diff
@@ -376,13 +402,21 @@
         },
         "enum": {
           "type": "array",
-          "items": {},
+          "items": {
+          },
           "minItems": 1,
-          "uniqueItems": true
+          "uniqueItems": false
```

Comment:
This is interesting - why would anybody ever want duplicate values in their enum?

Reply:
Great question, I'll ask!